tests/syncpoint/run.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+set -e
+
+CUR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=cdc.test
+SINK_TYPE=$1
+
+CDC_COUNT=3
+DB_COUNT=4
+
+function ddl() {
+    run_sql "DROP table IF EXISTS testSync.simple1"
+    sleep 2
```

Comment:
Why you add these `sleep 2` in the test, if having some special testing purpose, please add some comments.

Reply:
I want to make sure the ddl and dml operation can always execute during the test, so I add some `sleep 2` to get it. With this, every syncpoint will have different data, so the diff tool can judge the syncpoint's validity.